Automattic/wp-calypso
# Desktop: Enable guided-tours & Calypso site preview

Calypso site preview: Clicking the current selected site in the sidebar opens the preview layout, showing the site in an iframe.

![desktop-preview](https://cloud.githubusercontent.com/assets/215074/15867190/f3aa2580-2cd9-11e6-9767-ab90703740cb.gif)

Guided tours: When signing up, or adding a new site, the main guided tour displayed, which aims to provide some helpful onboarding. Currently in an abtest set at 50% of people.

![desktop-gt](https://cloud.githubusercontent.com/assets/215074/15867253/4ecf37ca-2cda-11e6-8636-acb5f3f6efc1.gif)


To test:
1. Set up the `wp-desktop` repo: https://github.com/Automattic/wp-desktop#getting-started--running-locally
2. Checkout the `calypso` submodule to HEAD of master
3. Check the site preview works on different types of sites (wp.com, jetpack, custom domains etc)
4. Add a new site, setting `localStorage.setItem( 'ABTests', '{"guidedTours_20160603":"guided"}' );` in the electron dev console.
5. Check the guided tour works

/cc @lsinger @mcsf 

Test live: https://calypso.live/?branch=add/guided-tours-and-preview-to-desktop
